pkg/controller.v1alpha3/suggestion/suggestion_controller_util.go

```diff
@@ -34,3 +35,43 @@ func (r *ReconcileSuggestion) reconcileService(service *corev1.Service) (*corev1
 	}
 	return foundService, nil
 }
+
+func (r *ReconcileSuggestion) deleteDeployment(instance *v1alpha3.Suggestion) error {
+	deploy, err := r.DesiredDeployment(instance)
+	if err != nil {
+		return err
+	}
+	realDeploy := &appsv1.Deployment{}
+	err = r.Get(context.TODO(), types.NamespacedName{Name: deploy.Name, Namespace: deploy.Namespace}, realDeploy)
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+	err = r.Delete(context.TODO(), realDeploy)
```

Comment:
Can you add add log here, that Suggestion deployment is deleted?

Reply:
It's done